### PR TITLE
Update the animals table with more datas.

### PR DESCRIPTION
**Pull request by Zel-hub7**
1. I have added a new column named species.
2. I have made calculations like avg weight, total number of animals, min and max weights and others...
3. I have made some transactions on the table according to the description given in the requirment.
4. I tried to screenshot every results.
 
transaction four
transaction second transaction
transaction three
transactiondb
transaction five
database 2nd

final

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -21,3 +21,100 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+--Verify that changes were made.
+SELECT * FROM animals;
+
+--COMMIT CHANGES
+
+COMMIT;
+
```
Comment (gemmen29) [inline queries.sql:88]: - [ ] Kindly add a transaction to delete all records in the animals' table, then roll back, then verify and take a screenshot of the results of your actions as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Now%2C%20take%20a,of%20your%20actions.).

File: queries.sql [hunk 2]
```diff
@@ -21,3 +21,100 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+--VERIFY IF THE CHANGES ARE PERMANENT
+
+SELECT * FROM animals;
+
```
Comment (gemmen29) [inline queries.sql:92]: - [ ] Kindly add a transaction that represent this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=of%20your%20actions.-,Inside%20a%20transaction%3A,Commit%20transaction,-Take%20a%20screenshot). Don't forget to Take a screenshot of the results of your actions.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Zel-hub7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @Zel-hub7 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the changes required ✔️ 
👍🏽 Professional PR with descriptive commit messages ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**